src/main/java/org/cbioportal/security/CancerStudyPermissionEvaluator.java

```diff
@@ -45,9 +45,11 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.*;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnExpression;
```

Comment:
Can you remove unused imports

Reply:
@Floris-Hyve I'm thinking the same thing, there are some unused imports, if we just need to change the way to create CancerStudyPermissionEvaluator, we don't need additional imports.

Reply:
Ah yes thanks for pointing out, removed them! This one seems unused also 
```import org.springframework.beans.factory.annotation.*;```